### ISSUE TITLE
Add timestamp to /count/{taskName}

### ISSUE THREAD

**Issue by tristen** [osmlab/to-fix-backend#10]
Currently a path like http://54.147.184.23:8000/count/unconnectedmajor/ returns

```
{"total":5096,"available":3646}
```

Ideally this also provides a timestamp so we can derive an x-axis for graphing this data. i.e

```
{[
    {"time":1426377600000,"total":5096,"available":3003},
    {"time":1426464000000,"total":5096,"available":3128},
    {"time":1426550400000,"total":5096,"available":3432},
    {"time":1426636800000,"total":5096,"available":3536}
]}
```

**Comment by aaronlidman** [comment]
Cool. `/count` was mostly meant for the bar graph:

![screen shot 2015-03-19 at 2 06 54 pm](https://cloud.githubusercontent.com/assets/58878/6737329/47148b46-ce41-11e4-8b8a-a25d50d8b7f6.png)

Adding history to this is going to take a little bit of work and something like a cron job. This data might fit best with https://github.com/osmlab/to-fix-backend/issues/3

**Comment by tristen** [comment]
Right - but I can derive that number from the latest date. This would power the brush graph below.

**Comment by aaronlidman** [comment]
Addressed with https://github.com/osmlab/to-fix-backend/pull/21
